test/test_commands.py

```diff
@@ -103,6 +103,12 @@ def test_log(self):
         expected = get_expected_output('log_limit')
         self.assertEqual(output, expected)
 
+    def test_log_mergeonly(self):
+        output = run_command(
+            'log', args=['--merge-only'], subfolder='immutable/tag')
+        expected = get_expected_output('log_merges_limit2')
```

Comment:
Why does the filename contain `limit2`? How about naming this `log_merge_only` instead?